rasterio/_env.pyx

```diff
@@ -57,6 +59,8 @@ code_map = {
 
 log = logging.getLogger(__name__)
 
+os.environ.setdefault("CURL_CA_BUNDLE", certifi.where())
```

Comment:
I dislike messing with the environment like this.

Reply:
I agree.

Reply:
There currently isn't a way around setting it at some point, so I think this is still a good option at this stage. If methods for setting the path for PROJ/GDAL are added to GDAL, then this could be avoided in the future..